examples/lock-app/cc13x2_26x2/BUILD.gn

```diff
@@ -0,0 +1,161 @@
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build/config/defaults.gni")
+import("//build_overrides/chip.gni")
+import("//build_overrides/ti_simplelink_sdk.gni")
+import("${ti_simplelink_sdk_build_root}/ti_simplelink_executable.gni")
+import("${ti_simplelink_sdk_build_root}/ti_simplelink_sdk.gni")
+
+assert(current_os == "freertos")
+
+project_dir = "${chip_root}/examples/lock-app/cc13x2_26x2"
+
+ti_simplelink_sdk("sdk") {
+  include_dirs = [
+    "${project_dir}",
+    "${project_dir}/main",
+    "${project_dir}/main/include",
+    "${chip_root}/src/platform/cc13x2_26x2",
+    "${project_dir}/main/include/",
+    "${project_dir}/main/",
+    "${chip_root}/third_party/openthread/repo/src/core",
+    "${ti_simplelink_sdk_root}/examples/rtos/${ti_simplelink_board}/thread/cli_mtd",
+    "${ti_simplelink_sdk_root}/source/ti/thread/rtos/${ti_simplelink_board}/thread/libmbedcrypto/config",
+  ]
+
+  sources = [
+    "${project_dir}/main/include/CHIPProjectConfig.h",
+    "${project_dir}/main/include/FreeRTOSConfig.h",
+  ]
+
+  defines = []
+  if (is_debug) {
+    defines += [ "BUILD_RELEASE=0" ]
+  } else {
+    defines += [ "BUILD_RELEASE=1" ]
+  }
+}
+
+source_set("openthread_platform") {
```

Comment:
Could this be moved into third_party/openthread/platforms ?

Reply:
didn't mean to click resolved, whoops.

The platform files depend on the SDK porting layer, I might be able to pull them out and have it just depend on the SDK. But I think this is clean.